brainstormforce/astra
# AST-2972 - Introducing new 'Disable Banner Area' metabox based option to control visibility of dynamic banner title area

### Description
- Have design requirement to get option to enable/disable banner title area
- - As like we have title enable/disable option in meta, but this only hides title not other parts from banner (like meta/breadcrumb etc)
- - So there is need to have this option which enable/disable page title banner area from individual meta
- - Considering 2 different page scenarios here, for home page & for normal page. When on home page normal page builder’s banner can take their place with transparent header and here don’t want dynamic banner title but on other page there is need of dynamic banner title.

### Screenshots
- https://share.getcloudapp.com/qGuqOYWv

### Types of changes
- New feature (non-breaking change)

### How has this been tested?
- Check with the option working or not. Override or not
- check for woocommerce product metabox once
- Check with customizer based global title area
- If customizer is off then there should not be this meta option
- check with banner layout 1 & 2 as well

### Checklist:
